.gitmodules

```diff
@@ -0,0 +1,3 @@
+[submodule "mit_tools"]
+	path = mit_tools
+	url = https://github.com/JustFrederik/mit_tools
```

Comment:
Is this necessary? Havent seen it being used.

Reply:
its a leftover part of a better translator lib and some other stuff that wasnt merged. i accedenlty pushed it. I removed the redundant file